components/team/Sidebar/index.tsx

```diff
@@ -32,7 +32,10 @@ const search = debounce((router: NextRouter, searchTerm: string) => {
     : undefined;
 
   searchParam
-    ? router.push(pathname, as, { query: { search: searchParam } })
+    ? router.push(
+        { pathname, query: as ? {} : { search: searchParam } },
+        as && { pathname: as, query: { search: searchParam } }
+      )
```

Comment:
This is _slightly_ unrelated to this PR. I broke some search functionality in my last set of changes, so this fixes it.